examples/systemd/fdo-client-linuxapp.service

```diff
@@ -5,7 +5,7 @@ After=network-online.target
 
```

Comment:
You did not actually remove the ConditionPathExists

Reply:
done